### PR TITLE
fix(pipeline): namespace not being detected

### PR DESCRIPTION
**Pull request by axieax**
namespace wasn't being detected during CodeArtifact npm publish

### CODE REVIEW

File: apps/hooks/pipeline/code-artifact-docker/index.ts [hunk 1]
```diff
@@ -25,6 +23,8 @@ exports.handler = async (
   statusCode: number;
   body: string;
 }> => {
+  console.log(JSON.stringify(event));
```
Comment (stevenp2) [inline apps/hooks/pipeline/code-artifact-docker/index.ts]: maybe remove this since it's for debugging purposes

**Review by stevenp2 — APPROVED** [review]
good 👍

File: apps/hooks/pipeline/code-artifact-docker/index.ts (hunk 1 above)
Comment (axieax) [reply apps/hooks/pipeline/code-artifact-docker/index.ts]: good catch :tropical_fish:

**Comment by axieax** [comment]
funny pr

**Pull request merged**